components/contribution-flow/index.js

```diff
@@ -388,7 +402,17 @@ class ContributionFlow extends React.Component {
   /** Steps component callback  */
   onStepChange = async step => {
     this.setState({ showSignIn: false });
-    this.pushStepRoute(step.name);
+    // To create an order we need a payment method to be set. This is normally set at final stage but for crypto flow we
+    // need to set this before the final step of the flow
+    if (this.props.paymentMethod === 'crypto') {
+      this.setState({ stepPayment: { key: 'manual', paymentMethod: { type: GQLV2_PAYMENT_METHOD_TYPES.CRYPTO } } });
+    }
+
+    // This checkout step is where the QR code is displayed for crypto
+    if (step.name === 'checkout') {
+      await this.submitOrder();
+    }
```

Comment:
Might be ridiculous code style comment but this is hurting my eyes. Line break here?

Reply:
You mean at the end of this `if` statement? I've added the line break.